.bin/check_and_load_xml.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+# Encode non UTF-8 files to UFT-8 for Postgres compatibility
+
+set -e
+
+database=${PGDATABASE:-ggircs_dev}
+input_file=$1
+encoded_file=${input_file/xmls_from_eccc/utf8_encoded_xmls}
+filename=$(basename -- $1)
+
+
+#convert xmls to utf-8 encoding
+convertToUFT8(){
+    CHARSET=$(file -I $input_file | awk -F "=" '{print $2}')
+    if [[ "$CHARSET" !=  'utf-8' ]]; then
+        echo "Converting $filename from $CHARSET to UTF-8"
+        iconv -f "$CHARSET" -t UTF-8 "$input_file" > ./data/eccc_xmls/utf8_encoded_xmls/"$filename"
+    fi
+}
+
+
+_psql() {
+  user=${PGUSER:-$(whoami)}
+  database=${PGDATABASE:-$database}
+  host=${PGHOST:-localhost}
+  port=${PGPORT:-5432}
+  psql -d "$database" -h "$host" -p "$port" -U "$user" -qtA --set ON_ERROR_STOP=1 "$@" 2>&1
+}
+
+
+# Check if schema is still the same
```

Comment:
With static paths like this, there's going to be a huge amount of noise in schema diffing. Can we use a purpose-built tool for this like [xmllint](http://xmlsoft.org/xmldtd.html) from libxml2 for schema validation? It's not the changes in the input schema we care about, but rather the file's ability to be processed by the parsers we've implemented. Most of those aren't looking at specific depths but rather class names and relationships along very specific traversal axes.

Reply:
Do we not want to check each node in an input XML against an index of historic nodes?

Reply:
Added xmllint